elide-core/src/main/java/com/yahoo/elide/core/filter/Predicate.java

```diff
@@ -43,6 +44,14 @@ public Predicate(PathElement pathElement, Operator op, List<Object> values) {
         this(Collections.singletonList(pathElement), op, values);
     }
 
+    public Predicate(PathElement pathElement, Operator op){
+        this(Collections.singletonList(pathElement), op, new ArrayList<Object>());
+    }
+
+    public Predicate(List<PathElement> path, Operator op){
+        this(path, op, new ArrayList<Object>());
+    }
+
```

Comment:
There is no constructor in Predicate class allow instantiate without `values`. And "ISNULL" "NOTNULL" operators are Nullary filters, so I think it's good to make them.